vuepress/docs/next/tutorials/create/pb/export-bundle-from-application.md

```diff
@@ -3,29 +3,24 @@ sidebarDepth: 2
 ---
 # Export and Publish a Bundle
 
-## Overview
-Use the bundler command to export a bundle of components from an existing Entando Application. An Entando Bundle can be used 
-- For the initial install of Entando components into an Entando Application
-- To migrate Entando components from one environment to another (e.g. Dev to QA)
-- To provide a template for building a new Entando Application
-- As the skeleton of an Entando solution
+The ent bundler command can export a bundle from an existing Entando Application to be utilized in the following ways: 
+- For the initial install of components into an Entando Application
+- To migrate bundles from one environment to another (e.g. Dev to QA)
+- To provide a template for building a new application
 
-This command generates the same bundle directory structure as an Entando project, including a bundle descriptor file.
+This tutorial illustrates how to use the ent bundler to export a git-based Entando Bundle. This package can be deployed to a running Entando instance or modified to a docker-based bundle to take advantage of Entando's latest composable methods. The procedure assumes you're using an Entando quickstart application. Otherwise, you may need to adjust specific URLs, credentials, namespaces, etc., for a custom application.
```

Comment:
"illustrates" is a bit inaccurate because there are no graphics or even a start-to-finish working example. describes?

"modified to" sounds odd; "converted to"

"Entando's latest composable methods" doesn't seem to fit. if that were the case, we should list them explicitly instead of with a vague reference; however, most of the benefits/changes consequent to the restructuring (for example, what alessandro listed in his demo slide) cannot be considered methods. 

can simplify the second sentence to close the distance between the subject and verb and avoid the pause of the last comma: "To use a custom application, you may need to..."

Reply:
2. modified is a great word choice, but you don't modify to something. modify to show/say/do etc., but modify to a state, not a thing. for example, modify to be a docker-based bundle works, albeit inelegantly

3. disagree... "modified to a docker-based bundle to take advantage of Entando's latest composable methods" is too vague to provide much value to the reader; also, there are several significant advantages to docker-based bundles, and this wording doesn't communicate that / limits and undersells the new structure and process. and i think we should be somewhat aware of using "methods" as a word choice, given someone versed in software will associate that word with "a programmed procedure that is defined as part of a class and included in any object of that class" and perhaps make some false assumptions. but generally speaking, pack+publish+deploy is how you create a bundle, and it's pretty much the only way to do it. whether the bundle is docker- or git-based, and regardless of what commands/tools are used.

4. i was only suggesting a quick edit to move a couple words

Reply:
@nshaw please weigh in on 3

Reply:
@nshaw we synced offline